ykview/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "ykview"
+version = "0.1.0"
+authors = ["bjorn3 <bjorn3@users.noreply.github.com>"]
```

Comment:
What happens if I were to mail that address? Does github forward it on?

Reply:
It will be ignored as far as I know.

Reply:
Probably not worth including an email address, if you don't wish to be emailed?

Reply:
It is included by default. According to the cargo reference you can fill in anything you want in `authors` though: https://doc.rust-lang.org/cargo/reference/manifest.html?highlight=author#the-authors-field